imagevector/images.yaml

```diff
@@ -223,7 +223,7 @@ images:
 - name: csi-snapshot-validation-webhook
   sourceRepository: github.com/kubernetes-csi/external-snapshotter
   repository: registry.k8s.io/sig-storage/snapshot-validation-webhook
-  tag: "v7.0.1"
+  tag: "v8.1.0"
```

Comment:
Did you test the snapshot validation with these changes ? I believe that there are some RBAC changes that we did in the other upgrade PRs (e.g. https://github.com/gardener/gardener-extension-provider-azure/pull/974)

Reply:
Done